.env

```diff
@@ -32,7 +32,7 @@ _APP_SMTP_PORT=1025
 _APP_SMTP_SECURE=
 _APP_SMTP_USERNAME=
 _APP_SMTP_PASSWORD=
-_APP_STORAGE_LIMIT=10000000
+_APP_STORAGE_LIMIT=30000000
```

Comment:
Do we want to keep this value? was it changed just for tests?

Reply:
We need this as we added test for large file, without this large file test will fail.